src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -84,11 +91,10 @@ public void toModelType_nullEmail_throwsIllegalValueException() {
     }
 
     @Test
-    public void toModelType_invalidAddress_throwsIllegalValueException() {
+    public void toModelType_blankAddress_returnsPerson() throws Exception {
```

Comment:
Would it better to declare that this function `throws IllegalValueException` (from `person.toModelType()`)? I see that there is a testcase above that just throws `Exception` but maybe `IllegalValueException` would be better?